pandas/io/sql.py

```diff
@@ -182,26 +182,28 @@ def execute(sql, con, cur=None, params=None):
 def read_sql_table(table_name, con, schema=None, index_col=None,
                    coerce_float=True, parse_dates=None, columns=None,
                    chunksize=None):
-    """Read SQL database table into a DataFrame.
+    """
+    Read SQL database table into a DataFrame.
 
     Given a table name and a SQLAlchemy connectable, returns a DataFrame.
     This function does not support DBAPI connections.
 
     Parameters
     ----------
-    table_name : string
+    table_name : str
         Name of SQL table in database.
-    con : SQLAlchemy connectable (or database string URI)
+    con : SQLAlchemy connectable or database URI str
```

Comment:
in this line we ideally would like to have just types (like type annotations). Not sure what the SQLAlchemy type would be, but for the database URI str, just leave `str`, and explain that is the database URI in the description of the parameter